cc/concurrent_selfplay.cc

```diff
@@ -105,7 +105,12 @@ DEFINE_double(policy_softmax_temp, 0.98,
               "policy_softmax_temp to encourage diversity in early play.\n");
 DEFINE_bool(restrict_in_bensons, false,
             "Prevent play in benson's regions after 5 passes have been "
-            "played.\n");
+            "played.");
+DEFINE_bool(allow_pass, true,
+            "If false, pass moves will only be read and played if there is no "
+            "other legal alternative. Disabling pass during the initial rounds "
```

Comment:
you can strike the second sentence here for simplicity.

Reply:
done